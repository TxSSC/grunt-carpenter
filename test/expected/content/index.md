<html>
<head>
  <title>Metadata here</title>
</head>
<body>
  <div class="content">
  <h2>Hello world!</h2>
<ul>
<li>Beep</li>
<li>Boop</li>
</ul>

</div>
</body>
</html>