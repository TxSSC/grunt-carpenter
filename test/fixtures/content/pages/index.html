---
Title: I have content
---

<h1>I am an html page and not generated</h1>