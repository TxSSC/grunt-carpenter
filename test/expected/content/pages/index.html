<html>
<head>
  <title>I have content</title>
</head>
<body>
  

<h1>I am an html page and not generated</h1>
</body>
</html>